frame/collective/src/tests.rs

```diff
@@ -115,25 +118,25 @@ impl frame_system::Config for Test {
 	type SystemWeightInfo = ();
 	type SS58Prefix = ();
 	type OnSetCode = ();
-	type MaxConsumers = frame_support::traits::ConstU32<16>;
+	type MaxConsumers = ConstU32<16>;
 }
 impl Config<Instance1> for Test {
 	type Origin = Origin;
 	type Proposal = Call;
 	type Event = Event;
-	type MotionDuration = MotionDuration;
-	type MaxProposals = MaxProposals;
-	type MaxMembers = MaxMembers;
+	type MotionDuration = ConstU64<3>;
+	type MaxProposals = ConstU32<100>;
+	type MaxMembers = ConstU32<100>;
```

Comment:
I think it is better to use MaxMembers and MaxProposal here instead of defining new value.
It makes more sense to reuse value. But anyway it is ok